### DISCUSSION TITLE
Class request: vehicle license plate, license plate number, and-or related entities

### DISCUSSION THREAD

**Discussion by jonathanvajda** [CommonCoreOntology/CommonCoreOntologies#387] [Ontology Terms and Class Schema]
There is a wide range of use cases where vehicles are described with respect to their vehicle plate (and the number), and this assists with entity resolution. This seems useful for manufacturing, commerce, government, university, and other contexts. I think this makes a good case for it being included in the common core rather than a lower-level ontology.

Just as with VIN (see the other issue https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/204 ) we need to be clear on what design pattern we are following here. Is this correct?
```
VehicleLicensePlate subClassOf cco:OneDimensionalBarcode
VehicleLicensePlateNumber subcClassOf cco:CodeIdentifier
```

For example:
```
VehicleLicensePlate _carries_ VehicleLicensePlateNumber
VehicleLicensePlate _has_text_value_ "ABC 1234"
VehicleLicensePlateNumber _designates_ VehicleLicense
```

My hesitation is that a vehicle plate number does not designate a vehicle, but rather a vehicle license. But people use it to designate a vehicle, indirectly. Is this correct?

**Comment by cameronmore** [comment]
Good suggestion, it seems that just about all countries require a license plate to travel on public roads, so it warrants consideration in the CCO.

My first intuition of a definition for Vehicle License Plate is `A One Dimensional Barcode which is designed to carry some information about the registration or legal status of the vehicle to which it is attached or part of`. I'm being verbose so we can trim, but these seem like the salient parts.

In America, you carry your license plate from car to car; in (parts of) Europe, the license plate stays with the car. This presents an interesting issue, but both have to do with the legal status of some vehicle to which the plate is attached, whether it stays with that car or may be moved from car to car.

**Comment by swartik** [comment]
Is a license plate a one-dimensional or two-dimensional encoding? The CCO definition of One-Dimensional Barcode states that its concretizations consist of one-dimensional symbols, as in [UPC barcodes](https://en.wikipedia.org/wiki/Universal_Product_Code). Digits are two-dimensional symbols. In fact, on the typical license plate, digits are stamped deeply enough to be three-dimensional, (Perhaps some expert on license plates can weigh in on whether the third dimension has an information-bearing use  or is simply a necessity of manufacturing.)

@cameronmore, sorry to add an obscure exception, but I'm not sure you always carry license plates from car to car in America. Once upon a time in California, if you bought an antique car you had the option to retain its antique license plate. That was in the 80's. I don't know the current law.

**Comment by cameronmore** [comment]
That's a good point about license plates being stamped 3d objects, I was thinking about that too as I typed out my response. I'm fine with making it 3d, but the distinction between 2d and 3d doesn't feel right in general here in this part of the hierarchy.

**Comment by jonathanvajda** [comment]
I was mistaken about dimensionality for a plate

Though bar codes for licence plate numbers are on registration documents.

**Comment by neilotte** [comment]
Moving to discussion for coordination with ICE/IBE strategy.
